websites/R/R-a-d.io/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "191388216509595648",
+		"name": "awooooool#2686"
+	},
+	"service": "R-a-d.io",
+	"description": {
+		"en": "An online radio focused on Japanese songs"
+	},
+	"url": "r-a-d.io",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Gax8mal.png",
```

Comment:
The logo should be 512x512 px. Also, this logo seems to be low quality or improperly upscaled?

Reply:
Yes, I'm not really well-versed in image manipulation and such, that was done with Microsoft PowerToys actually. Their site is actually open-source[](https://github.com/R-a-dio/site) including their [assets](https://github.com/R-a-dio/site/tree/develop/public/assets), but the assets is 10 years old and I can't find higher quality logo there.

I opted for their [text logo](https://i.imgur.com/nk9wY2E.png) instead, what do you think?